DegreeProject/Assets/Scripts/Player/Player.cs

```diff
@@ -330,54 +332,48 @@ private void ShouldDie()
             }
         }
     }
+}
 
-    #region Command Functions
+// Command & RPC's
+public partial class Player
+{
 
     [Command]
     private void CmdUpdateMovement()
     {
-        UpdateMovement();
         RpcUpdateMovement();
     }
 
     [Command]
     public void CmdOnShiftInputDown()
     {
-        OnShiftInputDown();
         RpcOnShiftInputDown();
     }
 
     [Command]
     public void CmdOnAttackInputDown()
     {
-        OnAttackInputDown();
         RpcOnAttackInputDown();
     }
 
     [Command]
     public void CmdSetDirectionalInput(Vector2 directionalInput)
     {
-        SetDirectionalInput(directionalInput);
         RpcSetDirectionalInput(directionalInput);
     }
 
     [Command]
     public void CmdOnJumpInputDown()
     {
-        OnJumpInputDown();
         RpcOnJumpInputDown();
```

Comment:
Bra att du har delat upp de olika RPC modes till att kalla olika metoder, det gör det absolut lättare att följa vart instruktionen kommer ifrån. 

Jag är intresserad av att veta hur det gått för dig att dyka in i nätverksdelen den här veckan. 
Som jag förstår det, så syncar Mirror automagiskt transformen utav spelaren. Om detta är fallet så bör remote spelaren aldrig utföra transform/rigidbody uppdateringar, då remote får en absolut position varje gång som transformen syncas ifrån local. Om du låter remote också uppdatera velocity/UpdateMovement så får du två input streams som säger olika, vilket kommer leda till en sk. "desync" mellan vart spelare 1 ser spelare 2, och vart spelare 2 ser sig själv.